File: src/main/java/no.nav.sbl/rest/domain/RSEvent.java
package no.nav.sbl.rest.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class RSEvent {
    public long id;
    public String veilederIdent;
    public String eventType;

}

File: src/main/java/no/nav/sbl/rest/domain/RSEvent.kt
package no.nav.sbl.rest.domain;

import no.nav.sbl.domain.ContextEvent

data class RSEvent(
    val veilederIdent: String,
    val eventType: String,
) {
    companion object {
        fun from(contextEvent: ContextEvent) = RSEvent(
            veilederIdent = contextEvent.veilederIdent!!,
            eventType = contextEvent.eventType!!,
        )
    }
}
